cli/lib/gateway.js

```diff
@@ -83,6 +83,27 @@ Gateway.prototype.start = (options,cb) => {
         env: options.env
     }
 
+    function getConfigFromRedis() {
```

Comment:
The function name says getConfigFromRedis, but the error messages talk about writing to redis.